spec/models/resource_spec.rb

```diff
@@ -18,4 +18,39 @@
   end
 
   it_behaves_like 'indexable', :resource
+
+  describe 'privacy' do
+    let(:resource) { create(:resource, privacy: :priv) }
+
+    describe 'after_commit on: [:create]' do
+      it "adds the #{name} to the search index after creation" do
```

Comment:
I think the naming here is a bit misleading. Can you update this spec name to be something like `"does not add the #{name} to the search index after creation"`? Or am I mistaken about what the spec is intended to do?